price-feeder/oracle/oracle.go

```diff
@@ -105,11 +105,7 @@ func (o *Oracle) Start(ctx context.Context) error {
 
 			o.lastPriceSyncTS = time.Now()
 
-			// XXX/TODO: We need to evaluate if this is the correct value to wait for
-			// between loop executions.
-			//
-			// Ref: https://github.com/umee-network/umee/issues/256
-			time.Sleep(10 * time.Millisecond)
+			time.Sleep(2250 * time.Millisecond)
```

Comment:
Came to this by looking at terra's timeout and testing it locally - pretty consistently doesn't run into the async broadcast issue, any lower and it does have a few once in a while. Want to keep this on the lower limit because we have retry logic in place & too long could skip voting periods

Reply:
ACK. Can we do two things:

1. Move this value into a `const` at the top of the file.
2. Add a comment above the `const` stating the rationale for the value picked. We can use what you just said above.

Reply:
Also, can you describe what the objective is? E.g. do we want the loop to trigger/tick X # of times per voting period...something along those lines?

Reply:
Twice per voting period, each with a vote and then a prevote (except for the first tick, which will wait until the next voting period) - will add comments about this and a const